src/test-error/index.js

```diff
@@ -0,0 +1,29 @@
+import TEMPLATES from './templates';
+
+const TAGS = [
+    //NOTE: we use tag names which differ of html tags for provide the one correspondence
```

Comment:
we use tag names that differ from html tags to ensure unambiguous correspondence